src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -373,6 +373,7 @@ class SharedLinkSection extends React.Component<Props, State> {
                     <span>
                         <FormattedMessage {...messages.linkShareOn} />
                         <Tooltip
+                            isTabbable={false}
```

Comment:
Should this component also be set to isTabbable=false? Your bugfix is for the toggle component, whereas this is for the toggle's label. How is the behavior changing here?